nwb.config.js

```diff
@@ -1,19 +1,53 @@
-const browsers = require('./test/browsers');
+const babel = require('babel-core');
 const externals = require('webpack-node-externals');
-const [ cmd ] = require('yargs').argv._;
+const fs = require('fs');
+const glob = require('glob');
+const mkdirp = require('mkdirp');
+const path = require('path');
+const rimraf = require('rimraf');
+const yargs = require('yargs');
+
+const browsers = require('./test/browsers');
+
+const [ cmd ] = yargs.argv._;
+const isBuild = cmd === 'build';
+const babelConfig = {
+  plugins: ['transform-react-jsx']
```

Comment:
We use it in tests, but don't need it for prod builds. The tests and bundles share the NWB babel config,